docs/workloads/batch/configuration.md

```diff
@@ -23,6 +23,10 @@ predictor:
   env: <string: string>  # dictionary of environment variables
   log_level: <string>  # log level that can be "debug", "info", "warning" or "error" (default: "info")
   shm_size: <string>  # size of shared memory (/dev/shm) for sharing data between multiple processes, e.g. 64Mi or 1Gi (default: Null)
+  dependency_path: # (optional)
```

Comment:
I think we should consider also adding a note about this to `python-packages.md` and `system-packages.md`, what do you think?